sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -822,6 +822,16 @@ class Column(val expr: Expression) extends Logging {
    */
   def like(literal: String): Column = withExpr { Like(expr, lit(literal).expr) }
 
+  /**
+   * SQL like expression. Returns a boolean column based on a SQL LIKE ESCAPE match.
+   *
+   * @group expr_ops
+   * @since 3.0.0
+   */
+  def like(literal: String, escapeStr: String): Column = withExpr {
+    Like(expr, lit(literal).expr, Option(escapeStr))
+  }
+
```

Comment:
I'm neutral on adding this. Is that enough to just use the SQL interface for this escape?

Reply:
I think it's OK. If this is not needed, I will remove it.

Reply:
Let's don't add it for now. I think we need this in SQL side only.

Reply:
OK.I will remove it.